app.rb

```diff
@@ -1,23 +1,37 @@
 require 'sinatra'
 require './lib/profile'
 
-set :application_id, 'EDIT_ME'
-set :secret, 'EDIT_ME'
+set :application_id, '36337108d63bb81c5f6595a505d3e8e4df44daa765bac25daaad51220892b8a4'
+set :secret, 'efcd3c84411ba2a1771221f3035f4f6cb517f36b4ab11847ca1b89478e4b8187'
 set :redirect_uri, 'http://localhost:4567/callback'
 set :site_url, 'http://wegotcoders.com'
 set :session_secret, 'secret'
 enable :sessions
 
-get '/primes' do
-  # TODO - Can we make this dynamic?
-  limit = 100
+def time_format(date)
+    require 'date'
+    new_date = Date.parse(date).strftime('%d/%m/%Y')
+    return new_date
+end
 
-  # TODO - add your prime number solution in the primes.rb file.
-  @sum = Primes.sum_to(limit)
+get '/primes' do    
+  @sum = Primes.sum_to(params[:prime_limit].to_i)
 
   erb :primes, :layout => :main
 end
 
+get '/multiples' do
+    @sumMultiples = Multiples.multiples(params[:multiples_limit].to_i)
```

Comment:
Ruby styling uses snake_case rather than camelCase... check out the style guide as it has lots of tips in there and helps you to spot mistakes https://github.com/bbatsov/ruby-style-guide